Readme and Changelog added (#1)
friedcucumber/wpf-themedesigner-app: README.md

<h1 align="center">
        <img src="media/ThemeDesignerLogo.gif" width=50% alt="DevExpress WPF Theme Designer" />
</h1>

<p align="center">
<a href="https://docs.devexpress.com/WpfThemeDesigner/118707/index">Documentation</a> | <a href="https://www.devexpress.com/Products/NET/Controls/WPF/Themes/">Website</a> | <a href="CHANGELOG.md">Changelog</a>
</p>

<p align="center">
<i>The DevExpress WPF Theme Designer simplifies theme customization and allows you to quickly <br /> develop custom themes by altering the base colors of existing DevExpress Themes.
</i>
</p>

----
## Table of Contents

* [Overview](#overview)
* [Features](#features)
* [Getting Started](#getting-started)
* [Get Help](#get-help)

## Overview

> The DevExpress WPF Theme Designer was designed to reduce the grunt work associated with custom app-wide theme development. It is a free standalone tool that allows you to generate custom themes by altering the base colors of existing DevExpress WPF Themes.

Reviewer: "the grunt work"??

Reviewer: Rewrite.

Author: Yep. I've copied this para from the website.
https://www.devexpress.com/products/net/controls/wpf/themes/

[![WPF Theme Designer YouTube video](media/youtube-video-thumbnail.png)](https://www.youtube.com/watch?v=c5HJj8UXp8g)

## Features

> * [Theme Customization](https://docs.devexpress.com/WpfThemeDesigner/119320/workspace/tools) <br/> The Theme Designer offers a number of features designed to improve your editing experience. Pick colors from any on-screen application, filter element colors and use other new tools to easily colorize any visual element of your application.

Reviewer: "designed to improve your editing experience."?

Reviewer: and why 'easily'?

Reviewer: ```suggestion
* [Theme Customization](https://docs.devexpress.com/WpfThemeDesigner/119320/workspace/tools) <br/> The Theme Designer allows you to pick colors from any on-screen application, filter element colors, and use other tools to paint your application's visual elements.
```

* [Palette](https://docs.devexpress.com/WpfThemeDesigner/400319/edit-colors) <br/> The latest DevExpress Themes contain a palette. 

> * [XAML Edit](https://docs.devexpress.com/WpfThemeDesigner/400298/edit-theme-in-xaml) <br/> The WPF Theme Designer empowers you to locate and modify control templates, styles and apply color presets. You can edit XAML files in the built-in Code View, or open the theme solution in Visual Studio.

Reviewer: empowers you? Why are you writing this?

> * Live Preview <br/> Flexibility is at the heart of our WPF Theme Designer. Once you've made your changes, you don't need to rebuild your entire project to view the result. A comprehensive set of previews and an ability to preview changes in control and demo views give you complete picture of your theme's appearance. Simply use the Designer's Preview window to evaluate the usability of your new Theme.

Reviewer: "is at the heart of our WPF Theme Designer"? Are you writing a technical document, or a marketing brochure?

Reviewer: Did you write this?

* [Import. Export. Share.](https://docs.devexpress.com/WpfThemeDesigner/118787/build-and-export-new-themes) <br/> The WPF Theme Designer ships with multiple options to save modified themes and retain them for later use. New themes can be built as assemblies or saved to a *.tdsln file. All changes made to individual color schemes remain consistent whenever you upgrade to a newer version of DevExpress WPF Controls.

## Getting Started

Download and install the WPF Theme Designer, then [get started](https://docs.devexpress.com/WpfThemeDesigner/118594/getting-started).

## Get Help

> You can ask any questions in our [Support Center](https://www.devexpress.com/Support/Center) or [Online Chat](https://www.devexpress.com/).

Reviewer: any questions? Do you mean 'any question'?
